testsuite/gnat2goto/tests/incomplete_dec/incomplete_dec.adb

```diff
@@ -0,0 +1,6 @@
+package body Incomplete_Dec is
+   procedure P (X : in out Partial_Dec) is
+   begin
+      X.A := X.A + 1;
+   end P;
```

Comment:
These tests don't include any code that's actually run by `cbmc` so it's a bit hard to say if the generated code is sensible just from this

Reply:
I guess that if it did not work, gnat2goto would crash when encountering incomplete declaration.

Reply:
True.  I was just testing gnat2goto really.